### DISCUSSION TITLE
Native support for `If-Modified-Since`

### DISCUSSION THREAD

**Discussion by FreshlyBrewedCode** [directus/directus#13128] [Feature Requests]
I was exited about #12231 because using the `Last-Modified` header seemed liked a simple way to cache files from the `/assets` endpoint. However, as far as I can tell directus does not support the `If-Modified-Since` header. Is this something worth implementing or is there another way to handle caching for files from the `/assets` endpoint?

**Comment by rijkvanzanten** [comment]
I definitely think that's worth adding 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
